package.json

```diff
@@ -18,9 +18,10 @@
     "dev": "tsdx watch",
     "build": "tsdx build",
     "prepublishOnly": "npm run build",
-    "pretest": "npm run lint && npm run build",
     "test": "tsdx test --env=jsdom",
-    "lint": "prettier --write --loglevel warn \"src/**/*\" \"stories/**/*\" \"test/**/*\" \"README.md\" \"docs/**/*\" && tsdx lint --fix \"src/**/*.{ts,tsx}\" \"test/**/*.{ts,tsx}\" \"stories/**/*.{ts,tsx}\"",
+    "prelint": "npm run prettier",
+    "lint": "tsdx lint \"src/**/*.{ts,tsx}\" \"test/**/*.{ts,tsx}\" \"stories/**/*.{ts,tsx}\"",
```

Comment:
Here is missing `--fix` option.

Reply:
I was having issues with the `-fix` command producing code that didn't compile. 

Maybe this was due to the missing configuration above.. I will try again with that config.

Reply:
Yea, it has to compile, otherwise our linting is broken.

Reply:
`--fix has been added back`